iroh-net/src/magic_endpoint.rs

```diff
@@ -135,6 +144,7 @@ impl Default for MagicEndpointBuilder {
             keylog: Default::default(),
             discovery: Default::default(),
             peers_path: None,
+            use_mdns: true,
```

Comment:
Keep it off by default for 1 release until we have some experience with it? It might cause trouble.

Reply:
yeah, also need to add that option on `iroh::Node`

Reply:
fixed